### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**
ReviewDB removed because the owner doesn't care if his platform is abused for spam reviews (had a private conversation via Discord)! On the contrary, he even finds it funny! I know that it was a lot of work to develop this plugin, but nobody needs such a platform!

### CODE REVIEW

**Comment by mantikafasi** [comment]
I think this pr is funny

File: src/plugins/pauseInvitesForever/index.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ function disableInvites(guildId: string) {
 export default definePlugin({
     name: "PauseInvitesForever",
     tags: ["DisableInvitesForever"],
-    description: "Brings back the option to pause invites indefinitely that stupit Discord removed.",
+    description: "Brings back the option to pause invites indefinitely that stupid Discord removed.",
     authors: [Devs.Dolfies, Devs.amia],
```
Comment (Scyye) [inline src/plugins/pauseInvitesForever/index.tsx:47]: Looks good over all. but you should add yourself to devs, ya know, cuz you changed smth :+1: then i think itll be ready to be merged :+1:

**Comment by sunnniee** [comment]
lgtm

**Comment by Vendicated** [comment]
?

**Comment by Justman100** [comment]
@Vendicated

Yeah, just close PRs that removes a plugin for an unnecessary trash platform! So you support that shit?

**Comment by verticalsync** [comment]
> @Vendicated
> 
> Yeah, just close PRs that removes a plugin for an unnecessary trash platform! So you support that shit?

He's a moderator of ReviewDB, and copypastas can be removed by the users themselves if they really don't want it, or can opt-out entirely of ReviewDB.

**Comment by Vendicated** [comment]
be polite and follow the code of conduct or you will be blocked from the repo. if you have any issues with the moderation, you should bring them up in the right channel (e.g. reviewdb server or vencord support channel) instead of sending non-constructive pull requests like these.

**Pull request closed without merging**